components/style/properties/gecko.mako.rs

```diff
@@ -3784,13 +3784,20 @@ fn static_assert() {
         }
 
         self.clear_overflow_sides_if_string();
-        self.gecko.mTextOverflow.mLogicalDirections = v.second.is_none();
```

Comment:
I need to sort through how servo uses this property to figure that out.  We'd need to move the "use logical sides" boolean into the computed value or something.  Will look into this.